instance_skel_types.d.ts

```diff
@@ -197,6 +197,7 @@ export interface CompanionFeedbackBase<TRes> {
 	) => TRes
 	subscribe?: (feedback: CompanionFeedbackEvent) => void
 	unsubscribe?: (feedback: CompanionFeedbackEvent) => void
+	isVisible?: (config: { [id: string]: InputValue }) => boolean
```

Comment:
This signature looks wrong, and is missing one being added to the action type too

Reply:
Re: Signature - fixing ...
Re: Missing in action? Actually this like should be here at all (I forgot to remove it). `isVisible` is an option property, which is why I added the property to `CompanionInputField`. This should cover options in actions, feedbacks and configs.